src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcdCallbackController.java

```diff
@@ -48,6 +55,28 @@ public CallbackResponse attachToCase(
         }
     }
 
+    @PostMapping(path = "/create-case")
+    public CallbackResponse createCase(
+        @RequestBody CcdCallbackRequest callbackRequest,
+        @RequestHeader(value = "Authorization", required = false) String idamToken,
```

Comment:
any reason this is not required ? This would be required unless we are validating this at later point ?

Reply:
so that we can return 401/403 I think. Making it required will cause Spring Boot handle it and return 400

Reply:
This follows same path as previous callback - always respond with 200 notifying what's the error. If path of this has changed - let's address in separate ticket stating what changes need to be done, how to respond back to ccd